frontend/public/components/namespace.jsx

```diff
@@ -141,6 +141,29 @@ class PullSecret extends SafetyFirst {
   }
 }
 
+const ResourceUsage = requirePrometheus(({ns}) => <div className="co-m-pane__body">
+  <Heading text="Resource Usage" />
+  <div className="row">
+    <div className="col-sm-6 col-xs-12">
+      <Line title="CPU Shares" query={[
+        {
+          name: 'Used',
+          query: `namespace:container_spec_cpu_shares:sum{namespace='${ns.metadata.name}'}`,
+        },
+      ]} />
+    </div>
+    <div className="col-sm-6 col-xs-12">
+      <Line title="RAM" query={[
+        {
+          name: 'Used',
+          query: `namespace:container_memory_usage_bytes:sum{namespace='${ns.metadata.name}'}`,
+        },
+      ]} />
+    </div>
+  </div>
+  <Bar title="Memory Usage by Pod (Top 10)" query={`sort(topk(10, sum by (pod_name)(container_memory_usage_bytes{pod_name!="", namespace="${ns.metadata.name}"})))`} humanize={humanizeMem} metric="pod_name" />
```

Comment:
Those `query`'s are a bit harder to read. Wondering if there is a way how to make it easier.

Reply:
Yeah, just moving existing code here.

It might be possible to format them across lines.

Reply:
yeah, was thinking out loud... no need to do it now